external/batch-import.php

```diff
@@ -0,0 +1,63 @@
+<?php
+ini_set('display_errors', 1);
+if (!defined('XHGUI_ROOT_DIR')) {
+    require dirname(__DIR__) . '/src/bootstrap.php';
+}
+
+$sites      = new Xhgui_Sites();
```

Comment:
Totally agree, lately I just setup PHPStom to whatever coding style is used for the projects and keep pressing Ctl-Alt-L. :smile: